cfgov/teachers_digital_platform/jinja2/teachers_digital_platform/survey/results-9-12.html

```diff
@@ -24,7 +24,7 @@
             Your money journey results<span aria-hidden="true" style="display:none"
                                             class="initials-display">: <span class="initials-value"></span></span>
         </p>
-        <p class="h1">Congratulations, {{ overall.heading_html|safe }}</p>
+        <h1>Congratulations, {{ overall.heading_html|safe }}</h1>
```

Comment:
@mrclay Is this element the top-level <h1> for the entire page? If not, then <p class="h1">  would be the correct markup to make sure that the page hierarchy is intact and is Section 508 compliant. 

@dcmouyard Is this your understanding as well?

Reply:
Yep, it's the only h1.